ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/StackComponentStyle.kt

```diff
@@ -0,0 +1,38 @@
+package com.revenuecat.purchases.ui.revenuecatui.components.style
```

Comment:
Hmm thinking about this, should this also go in the `stack` package? Seems like a better place to keep the same types together, and it's how we have the `TextComponentView`/`TextComponentStyle`.

Reply:
Ohh I see we moved the styles in the base PR... I'm wondering if it would make sense to keep them together though... Personally, I think that's better to keep the related files together.